crates/rspack/tests/tree-shaking/webpack-innergraph-circular/expected/main.js

```diff
@@ -57,17 +57,9 @@ const exportCUsed = __webpack_exports_info__.C.used;
 Object.defineProperty(exports, "__esModule", {
     value: true
 });
-function _export(target, all) {
-    for(var name in all)Object.defineProperty(target, name, {
-        enumerable: true,
-        get: all[name]
-    });
-}
-_export(exports, {
-    x: function() {
-        return x;
-    },
-    y: function() {
+Object.defineProperty(exports, "y", {
```

Comment:
This is expected since only `y` is used. https://github.com/web-infra-dev/rspack/pull/3547/files#diff-e89ad6620d23d2400a7b5e57a3eb84c3c1be6abff869f87080c699429ce40f7bR10